Release/bin/x64/plugins/cyber_engine_tweaks/mods/AppearanceMenuMod/Modules/props.lua

```diff
@@ -1066,6 +1067,9 @@ function Props:SavePropPosition(ent)
     end
   end
 
+  -- auto-refresh of all props, user pushed "save" button
+  if calledBeforeSaving then return end
```

Comment:
Did you run into a specific problem when not returning here?

The delayed update below does look kinda dodgy and it might be better to try to split creating a new blank preset off into a tick function but if there's a specific effect you discovered, it'd be useful to know!

Reply:
From what I saw in the code, updating a prop would also save the preset — yet saving the preset would not save potentially pending prop changes. I've added a loop through all props when the user pushes the button to save any changes, and I really didn't want to have an async save action pending from inside the loop :D